node/Makefile

```diff
@@ -389,8 +389,17 @@ run-k8s-apiserver: stop-k8s-apiserver run-etcd
 stop-k8s-apiserver:
 	@-docker rm -f st-apiserver
 
+GINKGO = ginkgo
+
+UT_PACKAGES_TO_SKIP?=pkg/lifecycle/startup,pkg/allocateip,pkg/status
+.PHONY: ut
+ut: CMD = go mod download && $(GINKGO) -r
```

Comment:
I don't think we need `go mod download`.

We can also get rid of the layers of variables in use here - let's just do this:

```
ut:
  $(DOCKER_GO_BUILD) sh -c '$(GIT_CONFIG_SSH) ginkgo -r -skipPackage` 
```